assets.src/src/redmine-tracky/controllers/timer-controller.ts

```diff
@@ -56,18 +55,28 @@ export default class extends Controller {
         }
     }
 
-    private timeDifference(): string {
-        const startDateTime = this.convertToDateTime((this.startTarget as HTMLInputElement).value);
-        const endDateTime = this.convertToDateTime((this.endTarget as HTMLInputElement).value);
+    private timeDiffToString(timeDiff: TimeDiff) {
+        const sign = (timeDiff.minutes < 0 || timeDiff.seconds < 0) ? '-' : '';
+    
+        return sign + ['hours', 'minutes', 'seconds']
+            .map((v) => timeDiff[v as keyof TimeDiff])
+            .map((v) => Math.abs(Math.floor(v)))
+            .filter((v, i) => i !== 0 || v !== 0) // Remove hours if zero
```

Comment:
attention....did you conciously switch the filter? i think you should first filter and then round, but maybe that is what you want?

Reply:
i did it because otherwise the `v !== 0` wouldn't work because it's not rounded. e.g. `0.01` instead of `0`